tools/nni_cmd/nnictl_utils.py

```diff
@@ -736,3 +739,162 @@ def search_space_auto_gen(args):
         print_warning('Expected search space file \'{}\' generated, but not found.'.format(file_path))
     else:
         print_normal('Generate search space done: \'{}\'.'.format(file_path))
+
+def save_experiment(args):
+    '''save experiment data to a zip file'''
+    experiment_config = Experiments()
+    experiment_dict = experiment_config.get_all_experiments()
+    if args.id is None:
+        print_error('please set experiment id.')
+        exit(1)
+    if args.id not in experiment_dict:
+        print_error('Cannot find experiment {0}.'.format(args.id))
+        exit(1)
+    print_normal('Saving...')
+    nni_config = Config(experiment_dict[args.id]['fileName'])
+    logDir = os.path.join(os.path.expanduser("~"), 'nni-experiments', args.id)
+    if nni_config.get_config('logDir'):
+        logDir = os.path.join(nni_config.get_config('logDir'), args.id)
```

Comment:
better to use a function to generate `logDir`, this function can be put in a file which store constants or meta-datas. 'nni-experiments' can be seen as a type of metadata

Reply:
updated